### PR TITLE
added main structure of the project in order to run with VueJS 

### PR DESCRIPTION
**Pull request by EricDS-INFO**
I ask you for this PR if you agree, ill keep working on the front. 
My next steps will be:

-initialize html structure of the main page
-create a component for the main page
-create posible diferent components for the web front

### CODE REVIEW

**Comment by pabloderivia** [comment]
Perfecto 👌💪

**Comment by armandodecanha** [comment]
I think it's ok :)

File: package.json [hunk 1]
```diff
@@ -1,12 +1,26 @@
...
+		"dev": "parcel src/index.html",
+		"clean": "rm -rf .cache dist"
+	},
+	"alias": {
+		"vue": "./node_modules/vue/dist/vue.esm"
+	},
+	"keywords": [],
+	"author": "",
+	"license": "ISC",
+	"dependencies": {
```
Comment (ArcaDelDelirio) [inline package.json]: The dependencies must be only vue and his dependencies like vue-router or vuex or react

File: src/App.vue [hunk 2]
```diff
@@ -0,0 +1,14 @@
+<template>
```
Comment (ArcaDelDelirio) [inline src/App.vue]: Why you use Main and App components ?

File: package.json (hunk 1 above)
Comment (armandodecanha) [reply package.json]: I'll bend the knee.

**Pull request merged**